risc0/zkvm/src/receipt/segment.rs

```diff
@@ -215,7 +248,8 @@ fn decode_system_state_from_io<E: Elem + Into<u32>>(
 pub(crate) fn decode_receipt_claim_from_seal(
```

Comment:
```suggestion
pub(crate) fn decode_receipt_claim_from_seal_v1(
```

Reply:
Although my theory here is to not make any or minimal legacy changes ahead of deciding on the final shape and form the API takes to support v1/v2 side-by-side, or even if we want to support side-by-side at all.